integration-tests/src/test/resources/deploywebapp.py

```diff
@@ -1,22 +1,12 @@
 # Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
-
-import time as systime
-
 connect(sys.argv[1],sys.argv[2],sys.argv[3])
-deploy(sys.argv[4],sys.argv[5],sys.argv[6],upload='false',remote='false')
-#systime.sleep(15)
-
-#cd ('AppDeployments')
-#myapps=cmo.getAppDeployments()
- 
-#for appName in myapps:
-#       domainConfig()
-#       cd ('/AppDeployments/'+appName.getName()+'/Targets')
-#       mytargets = ls(returnMap='true')
-#       domainRuntime()
-#       cd('AppRuntimeStateRuntime')
-#       cd('AppRuntimeStateRuntime')
-#       for targetinst in mytargets:
-#             curstate4=cmo.getCurrentState(appName.getName(),targetinst)
-#             print '-----------', curstate4, '-----------', appName.getName()
\ No newline at end of file
+appName=sys.argv[4]
+appPath=sys.argv[5]
+clusterName=sys.argv[6]
+print 'Deploying application[%s] to the Cluster[%s] located @ [%s] ' %(appName,clusterName,appPath)
+print 'Checking the server status on the Cluster[%s] ' %(clusterName)
+domainRuntime()
+state(clusterName,'Cluster')
```

Comment:
is the check is only for debugging purposes? or deploy needs to be aborted when cluster state is not in the expected state.

Reply:
it is just for debugging purpose, I used code from Vanaja, she is investigating the deploy issue